r-package/grf/DESCRIPTION

```diff
@@ -29,6 +30,9 @@ Imports:
 RoxygenNote: 6.1.1
 Suggests:
     DiagrammeR,
-    testthat
+    testthat,
+    knitr,
```

Comment:
Is it necessary to add these suggests to the package, if they're just used by CI?

Reply:
It's there because of the `VignetteBuilder: knitr` field below (R CMD check fails without them), but I could remove VignetteBuilder, pkgdown::build_site() builds them anyways